geode-docs/developing/partitioned_regions/using_custom_partition_resolvers.html.md.erb

```diff
@@ -19,47 +19,64 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-By default, <%=vars.product_name%> partitions each data entry into a bucket using a hashing policy on the key. Additionally, the physical location of the key-value pair is abstracted away from the application. You can change these policies for a partitioned region. You can provide your own data partitioning resolver and you can additionally specify which members host which data buckets.
+By default, <%=vars.product_name%> partitions each data entry into a bucket using a hashing policy on the key. 
+Additionally, the physical location of the key-value pair 
+is abstracted away from the application.
+You can change these policies for a partitioned region by providing
+your own partition resolver.
+The partitioning can go further with a fixed-partition resolver
+that specifies which members host which data buckets.
 
 <a id="custom_partition_region_data__section_CF05CE974C9C4AF78430DA55601D2158"></a>
 **Note:**
-If you are colocating data between regions and custom partitioning the data in the regions, all colocated regions must use the same custom partitioning mechanism. See [Colocate Data from Different Partitioned Regions](colocating_partitioned_region_data.html#colocating_partitioned_region_data).
+If you are both colocating region data and custom partitioning,
+all colocated regions must use the same custom partitioning mechanism.
+See [Colocate Data from Different Partitioned Regions](colocating_partitioned_region_data.html#colocating_partitioned_region_data).
 
-<a id="custom_partition_region_data__section_1D7043815DF24308ABE4C78BFDFEE686"></a>
+The two steps of implementing the interface
+and configuring the regions are described.
+These steps differ based on which partition resolver is used.
 
```

Comment:
Consider active voice here, maybe something like:

To custom-partition your region data, follow two steps:
- implement the interface
- configure the regions
These steps differ based on which partition resolver you use.

Reply:
Thanks Dave!  I made these changes.